asf_core_data/getters/mcs_getters/get_mcs_installations.py

```diff
@@ -4,11 +4,30 @@
 
 import pandas as pd
 import os
+import re
 
-from asf_core_data.getters.data_getters import load_s3_data, get_most_recent_batch_name
+from asf_core_data.getters.data_getters import (
+    s3,
+    load_s3_data,
+    get_most_recent_batch_name,
+)
 
 from asf_core_data.config import base_config
```

Comment:
I think the following should work (because of what's in `__init__.py`):
```suggestion
from asf_core_data import base_config
```

Reply:
Tested this and got Import Error so keeping original